src/test/test_namespaces_pattern.py

```diff
@@ -233,7 +209,8 @@ def test_ext_segment_pattern(self):
             "invalid.segment.",  # ends with a dot
             "invalid.segment-",  # ends with a dash
             "invalid/segment",  # slash not allowed
-            "com.example##foo",  # valid namespace with hash
+            "com.example##foo",  # hashes not allows
+            "com.example#foo",  # hashes not allows
```

Comment:
We are testing `ext-seg = bcp47 / "." x-name / translation` here, correct? Then, 213 should fail on missing dot, not on hashes. @bernhardreiter What am I missing?

Reply:
This testing function is testing on
```abnf
fragment-seg = 1*ALNUMLOW *( SPECIALCHAR 1*ALNUMLOW )
```
as assigned in src/ssvc/utils/patterns.py to EXT_SEGMENT_PATTERN. It seemed to be the better fit. The question is if that testing of subpatterns is actually helpful or not. And if not, if it should be removed. I've kept the tests as close a possible to the original test, so the review can see how the new pattern will fare towards the old tests. I suggest to keep it this way, but think about a cleaning overhaul after this PR.